### PR TITLE
Work on #62 - New mailbox

### PR DESCRIPTION
**Pull request by Lineflyer**
- Removed old support channels
- Add link to new Facebook page and new mailbox

- Correct URL for openwig source

Edit:
- Clean up Readme.md

### CODE REVIEW

File: src/main/assets/en_about.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
 <br/>&nbsp;&nbsp;<b>OpenWIG</b>
 <br/>&nbsp;&nbsp;&nbsp;&nbsp;matejcik
 <br/>&nbsp;&nbsp;&nbsp;&nbsp;
-<small><a href="https://code.google.com/p/openwig/">https://code.google.com/p/openwig</a></small>
+<small><a href="https://github.com/matejcik/openwig">https://github.com/matejcik/openwig</a></small>
```
Comment (bekuno) [inline src/main/assets/en_about.html]: I would directly link to https://github.com/cgeo/openwig

**Comment by bekuno** [comment]
> 
> 
> now the openwig link points to our branch, maybe you can link the
> 
> > Original author: **matejcik**
> 
> to https://github.com/matejcik instead?

I think we should use here our active repo.
There we need a README with a brief historic description.

File: README.md [hunk 2]
```diff
@@ -29,31 +29,13 @@ Such a procedure is [described in the c:geo wiki](https://github.com/cgeo/cgeo/w
 
 An overview of released beta and production versions and their changelog can be found on the [release page of this repository](https://github.com/cgeo/WhereYouGo/releases).
 
-# Old README.md
```
Comment (bekuno) [inline README.md]: If you remove the "Old README" section then you should change in all sections :-)

Comment (Lineflyer) [reply README.md]: Oh yes. Too tired. Will continue tomorrow.

File: README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ The  README.md of the old repository is attached at the end of this file for you
 ## Contact
 As there are no specific support and contact channels for this project yet, please use the issue tracker of this repository to get in contact with the development community for the time being.
```
Comment (bekuno) [inline README.md]: "no specific support and contact channels" - should we take a better wording now?
Maybe also refer to FB or so?

File: README.md [hunk 4]
```diff
@@ -25,410 +25,20 @@ to be completed
...
-		<li>added Zone border in both Locus and vector maps</li>
-		<li>added "Cancel" button when starting wherigo</li>
-		<li>fixed starting a new game</li>
-		<li>fixed encoding in "About application"</li>
-		<li>option to display WhereYouGo icon in the statusbar while the application is running</li>
-		<li>changed cartridge loading message</li>
-		<li>logging messages are appended to existing log file</li>
-	</ul>
-</p>
+An overview of released beta and production versions and their changelog can be found on the [release page of this repository](https://github.com/cgeo/WhereYouGo/releases).
```
Comment (bekuno) [inline README.md]: I would also refer to https://github.com/biylda/WhereYouGo/releases for older releases.

File: README.md (hunk 3 above)
Comment (Lineflyer) [reply README.md]: This readme only targets developers not primarily the community.

Within the app I referenced to FB already.

File: README.md (hunk 4 above)
Comment (Lineflyer) [reply README.md]: Should be obsolete now, that we have the most recent old version in our releases.

**Comment by Lineflyer** [comment]
Edited according to review and squashed.

**Pull request merged**